src/pages/costModels/createCostModelWizard/index.tsx

```diff
@@ -333,7 +349,7 @@ class CostModelWizardBase extends React.Component<Props, State> {
             name: this.state.name,
             type: this.state.type,
             description: this.state.description,
-            markup: this.state.markup,
+            markup: this.state.isDiscount ? '-' + this.state.markup : this.state.markup,
```

Comment:
I'm a little confused that you're setting `markup` for the context, but not `isDiscount`. On line 74, I see that you deconstruct `isDiscount` like so:

<img width="819" alt="Screen Shot 2021-07-29 at 10 28 28 AM" src="https://user-images.githubusercontent.com/17481322/127510367-4d2d7af7-1424-4741-9417-087cc20ef9a3.png">

Why is the context different in each case?

Reply:
Not setting isDiscount, but using the value(line 81) that has already been set in context.